File: figureGeometricphp/php/square.php

<?php


class Square extends FigureGeometric {
    private ?float $side;
    private ?float $calculate = null;

    public function __construct(float $side) {
        parent::__construct("square", "cuadrado");
        $this->side = $side;
    }

    public function getSide(): ?float {
        return $this->side;
    }

    public function getCalculateArea(): ?float {
        $this->calculate = $this->getSide() * $this->getSide();
        return $this->calculate;
    }
}
?>

File: figureGeometric/square.java
public class square extends figureGeometric {
    private final double side;
    private double calculate;

    public square(double side) {
        super("Square", "Cuadrado");
        this.side = side;
    }

    public double getSide() {
        return this.side;
    }

    @Override
    public double calculateArea() {
        this.calculate= this.side*this.side;
        return this.calculate;
    }
}
